etcdadm/etcdadm

```diff
@@ -525,11 +525,7 @@ member_restore_from_local_snapshot() {
 }
 
 member_env_file() {
-  local name
-  local env_file
-  name=$(member_name)
-  env_file=$(config_state_dir)/${name}.env
-  echo "${env_file}"
+  echo ${ETCDADM_MEMBER_ENV_FILE:-$(config_state_dir)/etcd-member.env}
```

Comment:
can't work on it right now :( feel free to close this one and open new or just keep editing this until you happy. As long as bug is fixed, I dont care who fixed it :))